src/richie/apps/core/context_processors.py

```diff
@@ -32,13 +33,23 @@ def site_metas(request):
             "domain": site_current.domain,
             "web_url": f"{protocol:s}://{site_current.domain:s}",
         },
+        "SOCIAL": {
+            "registration_endpoint": getattr(
+                settings, "SOCIAL_AUTH_EDX_OAUTH2_REGISTRATION_ENDPOINT", ""
+            )
+            or reverse("admin:index")
+        },
         "FRONTEND_CONTEXT": json.dumps(
             {
                 "context": {
                     "csrftoken": get_token(request),
                     "environment": getattr(settings, "ENVIRONMENT", ""),
                     "release": getattr(settings, "RELEASE", ""),
                     "sentry_dsn": getattr(settings, "SENTRY_DSN", ""),
+                    "oauth2_whoami": getattr(
+                        settings, "SOCIAL_AUTH_EDX_OAUTH2_WHOAMI_ENDPOINT", None
+                    ),
+                    "allowed_hosts": getattr(settings, "ALLOWED_HOSTS", []),
```

Comment:
I don't know if it's something safe to expose this setting.

Reply:
As mentionned above, I will remove `oauth2_whoami` endpoint from the context to pass it directly in props.